tensorboard/webapp/plugins/npmi/views/embeddings/embeddings_component.scss

```diff
@@ -65,6 +70,14 @@ limitations under the License.
   justify-content: center;
 }
 
+.run-divider {
+  content: '';
```

Comment:
What is the content you want to put here later?
It's not clear to me how you want to use this element. (From the naming I thought it is just a line to divide the run list below.  If my guess is correct, why not just use border attribute?)

Reply:
Yes, this is just a line for dividing things. This has been used like that in `main_component` as well. I guess either change it everywhere or do it the same, no? What do you suggest?

Reply:
I searched a bit but can't find it in `main_component`. Do you mean `grabber`? [link](https://github.com/tensorflow/tensorboard/blob/master/tensorboard/webapp/plugins/npmi/views/main/main_component.scss#L73)